be/src/util/jsonb_utils.h

```diff
@@ -191,7 +191,11 @@ class JsonbToJson {
             if (iter->klen()) {
                 string_to_json(iter->getKeyStr(), iter->klen());
             } else {
-                os_.write(iter->getKeyId());
+                if (iter->getKeyId() == 0) {
```

Comment:
is key == 0 special ? if special add comment to explain

Reply:
Yes, we do not use keyid before , just use keystr for jsonkey, and se I make keyid == 0 is special for empty key

Reply:
maybe use sMaxKeyId for safe and readable, 0 is not too special

Reply:
OK I will try